### ISSUE TITLE
No shift-click and ctrl-shift-click

### ISSUE THREAD

**Issue by bochen415** [jakimfett/Werkbench#10]
As title but I see you are already working on this so this ticket is just a reminder.

**Comment by bochen415** [comment]
I was really not specific enough when posting this.
I meant shif-clicking in the output item slot to craft a whole stack or to craft as many items as the amount of input items allow and transport it directly to the player's inventory.
By ctrl+shift+click I meant doing it in the crafting grid (recipe input) to get rid of only one item type from there and drop it in the player's inventory.

**Comment by jakimfett** [comment]
Awesome, that's definitely in my plans, probably not for v1.3, but it might get rolled in with the v1.3.x updates.

**Comment by 4rz0** [comment]
Any news on this?
This (and missing NEI-support, #12 ) are the things that unfortunately make the Werkbench pretty useless for us.

**Comment by jakimfett** [comment]
Yes and no, @4rz0. I've been doing a huge rewrite of the backend, which is why there hasn't been a new release in months. The rewrite will make stuff like shift clicking possible, and in some cases, easy.

NEI support is coming in v1.3.x, that much I can promise. I don't have any ETA, but it's in progress.

**Comment by Zsashas** [comment]
Question: why would Ctrl be needed for removing items from the grid? Normal Shift-click should be enough.
